packages/kbn-cli-dev-mode/src/base_path_proxy_server.ts

```diff
@@ -21,8 +20,7 @@ import { DevConfig, HttpConfig } from './config';
 import { Log } from './log';
 
 const ONE_GIGABYTE = 1024 * 1024 * 1024;
-const alphabet = 'abcdefghijklmnopqrztuvwxyz'.split('');
```

Comment:
why there is no `s`?

Reply:
And there are two `z`s 🤔 .
I thought at first the `s` missing was due to the short URL template `/r/s/{slug}`, but then `r` is here.

Reply:
maybe the implemented version wants to omit words like s**, s** or s**  (censored)

Reply:
like `a**` lol  fairly high chances of that happening

Reply:
TIL, `s` is the source of all evils 🤷🏼

Reply:
@spencer gave a perfect reason for there not being `s`: https://github.com/elastic/kibana/pull/5843#discussion_r49243059